### PR TITLE
add sendPasswordResetEmail function to email authentication

### PR DESCRIPTION
**Pull request by shnupta**
Tested on both android and ios.

I'll add the docs and example later tonight.

Also I'm not sure whether my reject in the ios implementation is correct - should I just use the string Reject method?

### CODE REVIEW

**Comment by LuisRodriguezLD** [comment]
looks great! can you add the docs please?

**Comment by LuisRodriguezLD** [comment]
Just a button is enough and don’t forget the docs

File: AuthExample/MainView.js [hunk 1]
```diff
@@ -143,6 +143,15 @@ var checkEmailVerification = function() {
...
 
+var sendPasswordResetEmail = function() {
+    EAuth.sendPasswordResetEmail(userEmail.value)
+    .then(function(response) {
+        console.log("Success! " + response);
+    }).catch(function(error) {
+        console.log("Error: " + error);
+    });
+}
+
```
Comment (LuisRodriguezLD) [inline AuthExample/MainView.js]: Can you remove the extra spacing?

File: AuthExample/MainView.ux [hunk 2]
```diff
@@ -10,21 +10,26 @@
...
         <Page Name="lobbyPage">
-            <StackPanel Width="92%" Height="460" Padding="20">
+            <StackPanel Width="92%" Height="480" Padding="20">
                 <Rectangle Layer="Background" CornerRadius="10" Color="#fff"/>
                 <Text Value="{currentPageTitle}" Alignment="Center" FontSize="35" TextColor="#2586EC"/>
                 <Text Value="{lobbyStatusText}" Alignment="Center" Margin="0,0,0,20"/>
                 <Basic.TextInput Width="100%" Value="{userEmailInput}" ux:Name="userName" Alignment="Center" PlaceholderText="Email Address" PlaceholderColor="#bbb" />
                 <Basic.TextInput Width="100%" Value="{userPasswordInput}" IsPassword="true" Alignment="Center" PlaceholderText="Password" PlaceholderColor="#bbb" />
 
+
```
Comment (LuisRodriguezLD) [inline AuthExample/MainView.ux]: this too please

File: AuthExample/MainView.ux [hunk 3]
```diff
@@ -10,21 +10,26 @@
...
+                <Basic.Button Text="Send Password Reset Email" Clicked="{sendPasswordResetEmail}" />
                 <NativeViewHost >
                     <Firebase.Authentication.Google.GoogleButton Margin="0,20,0,0"/>
                 </NativeViewHost>
                 <NativeViewHost >
                     <Firebase.Authentication.Facebook.FacebookButton Margin="0,20,0,0"/>
                 </NativeViewHost>
+
+                
+
```
Comment (LuisRodriguezLD) [inline AuthExample/MainView.ux]: and these three please

File: AuthExample/MainView.js (hunk 1 above)
Comment (LuisRodriguezLD) [inline AuthExample/MainView.js]: this line

**Comment by shnupta** [comment]
Thanks! 🙂

**Comment by LuisRodriguezLD** [comment]
Thank _you_!

**Pull request merged**